web/profiles/custom/os2loop/modules/os2loop_messages/os2loop_messages.module

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/**
+ * @file
+ * Hooks for os2loop messages.
+ */
+
+use Drupal\Core\Entity\EntityInterface;
+
+/**
+ * Implements hook_entity_insert().
+ */
+function os2loop_messages_entity_insert(EntityInterface $entity) {
+  $helper = \Drupal::service('os2loop_messages.helper');
```

Comment:
For consistency with the rest of our modules, the body of this hook implementation should be moved to an `entityInsert`, say, function in the helper and then we just call this from `os2loop_messages_entity_insert`, i.e.

```
function os2loop_messages_entity_insert(EntityInterface $entity) {
  Drupal::service('os2loop_messages.helper')->entityInsert($entity);
}
```

Reply:
Added helper functions